plugin/src/main/java/io/jenkins/plugins/casc/core/JenkinsConfigurator.java

```diff
@@ -56,11 +58,24 @@ protected Jenkins instance(Mapping mapping, ConfigurationContext context) {
                 .setter( noop() ));
 
         // Override "nodes" getter so we don't export Nodes registered by Cloud plugins
-        Attribute.<Jenkins,List<Node>>get(attributes, "nodes").ifPresent(a ->
-            a.getter(j -> j.getNodes().stream()
```

Comment:
can you still export the nodes after this change?
This getter was there to stop the cloud nodes showing up in exports

Reply:
Aaaaaa.. thats why it was never called.. I will get it back..